src/rpc/server.cpp

```diff
@@ -138,8 +133,7 @@ UniValue ValueFromAmount(const CAmount& amount)
     int64_t n_abs = (sign ? -amount : amount);
     int64_t quotient = n_abs / COIN;
     int64_t remainder = n_abs % COIN;
-    return UniValue(UniValue::VNUM,
-            strprintf("%s%d.%08d", sign ? "-" : "", quotient, remainder));
+    return UniValue(UniValue::VNUM, strprintf("%s%d.%08d", sign ? "-" : "", quotient, remainder));
```

Comment:
Not sure if there's any specific guideline on returns or breaking arguments onto other lines, but I find it more readable to be able to break arguments onto separate lines. I personally think it's wise to lean toward readability vs compactness.